app/email/templates/icons/heroIcon.tsx

```diff
@@ -0,0 +1,186 @@
+const HeroIcon = () => {
+  return (
+    <div className="flex items-center justify-center">
+      <div className="hidden lg:block">
```

Comment:
Please all assets should be in the public folder. Find a better way to include svg icons.